### PR TITLE
256 translate trusted committer chapter07

### PR DESCRIPTION
**Pull request by RJPlog**
Hab Kapitel 7 übersetzt. Ich bin nicht glücklich mit den häufigen Erwähnungen der "Contributors", wollten wir ja nicht übersetzen, aber Contributors klingt für mich nicht gut. Hat jemand eine Bessere Idee? Contributoren ist ja auch nicht das gelbe vom Ei.

### CODE REVIEW

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Trotzdem sollte die Rolle des _Trusted Committers_ freiwillig übernommen werden, denn sie erfordert um erfolgreich zu sein, großen Zeitaufwand und Engagement. 
+
+Welche Kriterien gelten bei der Nominierungen von https://innersourcecommons.org/resources/learningpath/contributor/inde[Contributors] für die Rolle des _Trusted Committers_? Was benötigt man, um die Rolle des _Trusted Committers_ erfolgreich auszufüllen? Zuallererst müssen potentielle _Trusted Committer_ ihre tiefe technische Kompetenz während ihrer Arbeit in der Community unter Beweis stellen. Zusätzlich dazu müssen sie ihre Fähigkeit unter Beweis stellen, wirkunsvoll mit den Kollegen der Community zusammenzuarbeiten, und idealerweise genauso mit den Product Ownern und dem Management.
```
Comment (arnom-ms) [inline trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: link not correct, 

```suggestion
Welche Kriterien gelten bei der Nominierungen von https://innersourcecommons.org/resources/learningpath/contributor/index[Contributors] für die Rolle des _Trusted Committers_? Was benötigt man, um die Rolle des _Trusted Committers_ erfolgreich auszufüllen? Zuallererst müssen potentielle _Trusted Committer_ ihre tiefe technische Kompetenz während ihrer Arbeit in der Community unter Beweis stellen. Zusätzlich dazu müssen sie zeigen, dass sie wirkunsvoll mit den Kollegen der Community, und idealerweise genauso mit den Product Ownern und dem Management, zusammenzuarbeiten können.
```

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Auf die selbe Art müssen sie den Willen und die Geduld aufweisen, ihre Fähigkeiten zu nutzen und bewusst ihre Zeit spendieren, um _Contributors_ weiter zu qualifizieren. Schließlich erfordert die Erfüllung der Rolle des _Trusted Committers_ eine gewisse emotinale Reife, um mit stressigen sozialen Situationen umzugehen, die mit Sicherheit ab und zu aufkommen.
```
Comment (arnom-ms) [inline trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]:
```suggestion
Auf die selbe Art müssen sie den Willen und die Geduld aufweisen, ihre Fähigkeiten zu nutzen und bewusst ihre Zeit spendieren, um _Contributors_ weiter zu qualifizieren. Schließlich erfordert die Erfüllung der Rolle des _Trusted Committers_ eine gewisse emotinale Reife, um mit stressvollen sozialen Situationen umzugehen die mit Sicherheit ab und zu aufkommen.
```

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Contributors, welche diese Kritierien erfüllen, sind nach unserer Meinung gute Kandidaten für potentielle _Trusted Committers_.
+
+Die Rolle des _Trusted Committers_ wird nicht für alle Contributors attraktiv sein, bedeutet es doch, dass man weniger Zeit verbringt, zu selbst zu programmieren. Als _Trusted Committer_ vorgeschlagen zu werden kann sogar herabstufend empfunden werden oder als negatives Feedback bezüglich der eigenen Programmierfähgigkeiten aufgefasst werden. Aber das Gegenteil ist der Fall. Als _Trusted Committer_ nominiert zu werden, bedeutet normalerweise, dass man deine wertvollen Beiträge erkannt hat und in dir das Potential weiter zu wachsen und zu führen sieht. Die Rolle des _Trusted Committers_ gib dir mehr Einfluß auf die Entwicklung der Codebasis und macht dich definitiv zu einem vollständigeren Entwickler. Contributors zu erklären, wie die Software funktioniert, führt häufig zu neuen Erkenntnissen seitens des _Trusted Committers_ und hilft Möglichkeiten zu identifizieren, die Software weiter zu verbessern.
```
Comment (arnom-ms) [inline trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]:
```suggestion
Die Rolle des _Trusted Committers_ wird nicht für alle Contributors attraktiv sein, bedeutet es doch, dass man weniger Zeit verbringt, selbst zu programmieren. Als _Trusted Committer_ vorgeschlagen zu werden kann sogar herabstufend empfunden oder als negatives Feedback bezüglich der eigenen Programmierfähgigkeiten aufgefasst werden. Aber das Gegenteil ist der Fall. Als _Trusted Committer_ nominiert zu werden, bedeutet normalerweise, dass man deine wertvollen Beiträge erkannt hat und in dir das Potential weiter zu wachsen und zu führen sieht. Die Rolle des _Trusted Committers_ gib dir mehr Einfluß auf die Entwicklung der Codebasis und macht dich definitiv zu einem vollständigeren Entwickler. Contributors zu erklären, wie die Software funktioniert, die Rolle führt häufig zu neuen Erkenntnissen seitens des _Trusted Committers_, und hilft Möglichkeiten zu identifizieren die Software weiter zu verbessern.
```

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Ob es einer InnerSource Community einer oder mehrere _Trusted Committer_ gibt, hängt von der Größe und dem Risiko, welches mit ihrer Software verbunden ist, ab.
```
Comment (arnom-ms) [inline trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]:
```suggestion
Ob es einer InnerSource Community einen oder mehrere _Trusted Committer_ gibt, hängt von der Größe und dem Risiko, welches mit ihrer Software verbunden ist, ab.
```

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Die Rolle des _Trusted Committers_ ist zeitaufwändig, und nicht jeder ist gewillt oder hat die Möglichkeit diese Verpflichtung einzugehen. Aus diesem Grund nutzen mache Unternehmen ein Rotationssystem, in dem sich mehrere _Trusted Committer_ die Arbeitslast teilen, und diejenigen _Trusted Committer_, die gerade nicht an der Reihe sind, können sich ausschließlich technischen Themen widmen. Mehr als einen _Trusted Committer_ zu haben macht es auch leichter, wenn jemand das Unternehmen verlässt oder wenn jemand sich aus der Rolle in eine neue Aufgabe verändert. Für diesen Fall ist es wichtig, dass schon andere _Trusted Committer_ bereit stehen, die die Aufgaben übernehmen können und somit Kontinuität in der Gemeinschaft sicherstellen.
```
Comment (arnom-ms) [inline trusted-committer/de/07-becoming-a-trusted-committer.asciidoc:19]:
```suggestion
Die Rolle des _Trusted Committers_ ist zeitaufwändig, und nicht jeder ist gewillt oder hat die Möglichkeit diese Verpflichtung einzugehen. Aus diesem Grund nutzen mache Unternehmen ein Rotationssystem, in dem sich mehrere _Trusted Committer_ die Arbeitslast teilen, und diejenigen _Trusted Committer_, die gerade nicht an der Reihe sind, können sich ausschließlich technischen Themen widmen. Mehr als einen _Trusted Committer_ zu haben macht es auch leichter, wenn jemand das Unternehmen verlässt oder wenn jemand sich aus der Rolle in eine neue Aufgabe verändert. Für diesen Fall ist es wichtig, dass schon andere _Trusted Committer_ bereit stehen, die die Aufgaben übernehmen können um Kontinuität in der Gemeinschaft sicherstellen.
```

**Review by arnom-ms — COMMENTED** [review]
Some sugestions to change. THANKS fro translating the articles!

**Comment by RJPlog** [comment]
@arnom-ms thank you very much for your review. 

I incorporated all the changes, but sometimes don't know what is happening. At one place you found "Ob es einer InnerSource Community **einer** ...", correct would be "Ob es einer InnerSource Community **einen**... " But both  my local and my remote branch were showing already "Ob es einer InnerSource Community **einen**... ", so how could it be that you are seeing the wrong spelling? 

Second thing I don't understand, some of your suggestions I could "commit suggestion" by pushing a button, at other's the button is grey deposited and marked as outdated - do you know why this happens?

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+InnerSource Communities funktionieren nach denselben Prinzipien wie Open Source Communities, eines davon ist Meritokratie. In einer Meritokratie wird Ansehen basierend auf Talent, Anstrengung und Erreichtem erworben. Das bedeutet, dass das Ansehen und Privilegien, die mit der Rolle des _Trusted Committers_ einhergehen, verdient werden müssen. 
+Transparenz, ein weiterer Wert von Open Source, spielt ebenfalls eine wesentliche Rolle dabei, Talent, Anstrengung und Erreichtes in der gesamten Gemeinschaft sichtbar zu machen. 
+
+Der Prozess, offiziell ein _Trusted Committer_ zu werden, unterscheidet sich von Community zu Community und hängt davon ab, wo du persönlich in deiner Innner Source Reise stehst, und wird sich im Lauf der Zeit entwickeln. In neugegründeten Communities übernehmen die Gründer oft selbst die Rolle des _Trusted Committers_. Wenn die Community dann wächst, oder in größeren Communities, werden _Trusted Committer_ normalerweise nominiert oder von den https://innersourcecommons.org/resources/learningpath/contributor/index[Contributors] der Community gewählt.
```
[the remark this answers is not in the document]
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: done

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+Trotzdem sollte die Rolle des _Trusted Committers_ freiwillig übernommen werden, denn sie erfordert um erfolgreich zu sein, großen Zeitaufwand und Engagement.
```
[the remark this answers is not in the document]
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: done

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc (hunk 1 above)
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: done

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc (hunk 3 above)
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: done

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc (hunk 4 above)
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: strange thing, my local and my remote rep do not show the red part, there is already the green one visible?

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc (hunk 5 above)
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: done

File: trusted-committer/de/07-becoming-a-trusted-committer.asciidoc (hunk 2 above)
Comment (RJPlog) [reply trusted-committer/de/07-becoming-a-trusted-committer.asciidoc]: done

**Comment by arnom-ms** [comment]
Not sure what happens there, but it sounds like you hav emade the changes but not pushed? You need to do a git add.... followed by a git commit .... folled by a git push .... to get the changes to GitHub.

**Comment by RJPlog** [comment]
Any more suggestions? All the comments of arnom-ms are applied

**Comment by rrrutledge** [comment]
Not being a German speaker I don't have anything, but I can merge once there is a native speaker that signs off.

**Review by lenucksi — APPROVED** [review]
This German speaker signs off on it. Thanks @RJPlog, reads very nicely! 😃

**Comment by lenucksi** [comment]
An on the matter of a translation of "Contributor" I would get to "Beitragende" or something similar. But in the end I'm not a friend of Paketvermittlungsknoten, , Gruppenruf, Halden, Stapel, Fegelinienalgorithmen and other very much correct but in the ungoogable translations. Hence, in my book book "Contributor" is fine to me. I'm not totally opposed to something else though.

**Comment by rrrutledge** [comment]
Awesome!  Is anyone interested in making the pull request to _innersourcecommons.org_ to update for this new page?

**Pull request merged**